src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
Now it seems `Utils::String::unquote` should move to `Utils::Net::unquote`... what do you say?

Although `StringOption::unquote` behavior is the normally expected one, it's not preferable to use it in http parsing, because double quote is not allowed as value in most cases (so better strip it).